bem-css/style.css

```diff
@@ -76,7 +76,7 @@ body {
   color: #FFF;
 }
 
-.label__field {
+.input {
   display: block;
```

Comment:
Agree with wrong-named input, regular simple block 'label' shouldn't have any elements at all. 
But regular input shouldn't have so specific rules too!
We need to refactor this naming somehow, maybe `search__field` will be good idea, what do you think?